ember.md

```diff
@@ -341,9 +341,11 @@ order. [Don't Don't Override Init](https://dockyard.com/blog/2015/10/19/2015-don
 
 Models should be grouped as follows:
 
+* Services
```

Comment:
Thank you for also adding services here, since that was not explicit before either.